commands/sc.js

```diff
@@ -0,0 +1,185 @@
+const axios = require('axios')
+const config = require('config');
+const AsciiTable = require('ascii-table');
+const Discord = require('discord.js');
+let token;
+const airportRegex = /[A-Z]{4}/;
+const aircraftRegex = /[A-Z]{1,}[0-9]{1,}[A-Z]?/;
+
+module.exports = {
+    name: 'sc',
+    description: 'Interact with Shared Cockpit Requests',
+    help: true,
+    async execute(message, args) {
+        var data = {
+            'grant_type': 'client_credentials',
+            'client_id': 3,
+            'client_secret': config.get('auth.secret')
+        }
+        await axios.post('http://homestead.test/oauth/token', data)
```

Comment:
Think we're going to need some sort of `.env` for this

Reply:
Ah yeh, I'll set one up in the config